From b2b11ee5f4ee8b6e8b472ff878592176d5f31740 Mon Sep 17 00:00:00 2001
From: sam_liao <sam_liao@accton.com>
Date: Tue, 19 Jun 2018 10:50:03 +0000
Subject: [PATCH 2/2] Fit T600 HW ,Remove MMC to avoid "mmc0: Timeout waiting
 for hardware interrupt." timeout message 	modified:  
 arch/powerpc/configs/85xx/e6500rev2_defconfig

---
 arch/powerpc/configs/85xx/e6500rev2_defconfig | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/arch/powerpc/configs/85xx/e6500rev2_defconfig b/arch/powerpc/configs/85xx/e6500rev2_defconfig
index 53f97614d6a..be106678c4e 100644
--- a/arch/powerpc/configs/85xx/e6500rev2_defconfig
+++ b/arch/powerpc/configs/85xx/e6500rev2_defconfig
@@ -139,10 +139,10 @@ CONFIG_USB=y
 CONFIG_USB_EHCI_HCD=y
 CONFIG_USB_EHCI_FSL=y
 CONFIG_USB_STORAGE=y
-CONFIG_MMC=y
-CONFIG_MMC_SDHCI=y
-CONFIG_MMC_SDHCI_PLTFM=y
-CONFIG_MMC_SDHCI_OF_ESDHC=y
+# CONFIG_MMC is not set
+# CONFIG_MMC_SDHCI is not set
+# CONFIG_MMC_SDHCI_PLTFM is not set
+# CONFIG_MMC_SDHCI_OF_ESDHC is not set
 CONFIG_EDAC=y
 CONFIG_EDAC_MM_EDAC=y
 CONFIG_EDAC_MPC85XX=y
-- 
2.11.0

